shell.nix

```diff
@@ -16,6 +16,9 @@ let
   nomayastor_moth = "You have requested an environment without mayastor, you should provide it!";
   channel = import ./nix/lib/rust.nix { inherit sources; };
   mayastor = import pkgs.mayastor-src { };
+  # python environment for test/python
```

Comment:
is `test/python` the directory, should it be `tests/tests/bdd` ?

Reply:
Changed to `tests/bdd`.